site/app/controllers/pdf/PDFController.php

```diff
@@ -20,7 +20,7 @@ public function __construct(Core $core) {
      * @param $filename
      * @Route("/{_semester}/{_course}/gradeable/{gradeable_id}/pdf")
      */
-    public function showStudentPDF($gradeable_id, $filename, $path) {
+    public function showStudentPDF($gradeable_id, $filename, $path, $grader = null) {
```

Comment:
Did the changes to this file sneak in from another PR, or do they apply here?

Reply:
This is how, when a user selects a certain peer in the peer feedback tab, only pdf annotations from that peer are rendered. I think it makes sense to apply them here.